### ISSUE TITLE
Blade Flurry 6 stage calculations are incorrect

### ISSUE THREAD

**Issue by aggixx** [Openarl/PathOfBuilding#130]
A few issues here with the "6 Stages" and "Release at 6 Stages" settings for Blade Flurry.

1. The 6 stage calculation is showing 100% more attack damage when it should actually be 120% more. While you don't get the damage bonus of a stage as you gain it, once you reach 6 stages if you keep channeling you get 120% more damage.
2. "Release at 6 stages" setting seemingly does exactly what 1 stage does. You can calculate the damage multiplier of a 6 stage cycle like so:

`( 1 + 1.2 + 1.4 + 1.6 + 1.8 + 2.0 + 1 * 6  ) / 6 = 2.50`

So with that in mind there's two simple approaches that should be relatively easy to implement:
- 150% more attack damage
- 100% more attack speed (to emulate each attack hitting twice) and 25% more attack damage. In this way your average damage would truly be the average hit damage, but the APS may be a bit deceiving.

**Comment by aggixx** [comment]
Also, it may be useful to have a setting that shows the average damage (not DPS) of a release while at 6 stacks.

**Comment by Openarl** [comment]
1. Good point. I'll fix that in the next update.

2. The final strikes from Blade Flurry do benefit from the stage bonus; I've tested this myself. So the complete sequence of attacks deals 22.2x base damage. Since you won't be able to release and restart channelling at the exact moment you hit 6 stacks, I add an additional attack time to the estimate. That gives an approximate DPS multiplier of 3.2. Since it's just an estimate, I've rounded it down to 3, which is the value currently used by the program. 
I am aware that the current method doesn't correctly estimate the average damage, since it just multiplies the final DPS number. That won't be straightforward to fix, but I'll give it some thought.
